src/ResourceManager/Compute/Commands.Compute/Generated/Models/PSRunCommandInput.cs

```diff
@@ -0,0 +1,29 @@
+// 
```

Comment:
This should not be edited

Reply:
I have not touched this file.  I am going to do a full --hard reset on upstream and copy my changes in.  Something weird seems to have happened.